docs/whatsnew/3.0.rst

```diff
@@ -38,3 +38,56 @@ Full change log
 
 To see a detailed list of all changes in version v3.0, including changes in
 API, please see the :ref:`changelog`.
+
+
+Renamed/removed functionality
+=============================
+
+pytest plugins
+**************
+
+The following pytest plugins were previously provided as part of the Astropy
+core package but have now been moved to separate packages:
+
+* `pytest-remotedata <https://github.com/astropy/pytest-remotedata>`__
+* `pytest-doctestplus <https://github.com/astropy/pytest-doctestplus>`__
+* `pytest-openfiles <https://github.com/astropy/pytest-openfiles>`__
+
+Affiliated packages that make use of one or more of these plugins may install
+them individually using ``pip``, or they may install the Astropy testing
+package ``pytest-astropy``, which includes all of these plugins as
+dependencies. Packages that use Astropy's test runner will also need to install
+``pytest-astropy``. See :ref:`testing-dependencies` for more details.
+
+The ``config`` and ``display`` plugins are still provided as part of the
+Astropy core package, but they have moved from ``astropy.tests.pytest_plugins``
+to ``astropy.tests.plugins.config`` and ``astropy.tests.plugins.display``,
+respectively.
+
+Affiliated packages that currently access plugin decorators and other variables
+from ``astropy.tests.pytest_plugins`` will need to make updates, although
+backwards compatibility will be maintained in the meantime.
+
+Below is an outline of the required changes:
```

Comment:
This list is very valuable and should be move up to the actual testing doc, rather than be buried in "what's new". Instead what's new here can link to the relevant testing doc section for this.

Reply:
I actually disagree with this: I think it's not particularly relevant to most readers of the testing documentation, but it is important for affiliated package developers who need to know what changed in this Astropy release. But I can be persuaded otherwise if everyone else feels strongly about it.

Reply:
In my past experience with `config` refactoring back in 0.4, I never looked in What's New because I already knew what was new. Instead, I went straight to `astropy.config` section for the transition doc. But let's wait a bit for inputs from others.

Reply:
I would suggest to keep it; for my own affiliated package, I can use any help I can get in finding what things change!